README.md

```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
 The vc-http-api repository is structured with two main sections:
 
 - [docs](./docs/) which contains the VC HTTP API spec itself in [OpenAPI](https://swagger.io/specification/) (formerly known as Swagger) form as well as related documentation to assist in building and testing a compliant implementation of the API spec
-  * Architecture for the issuer model is discussed in the [architecture document](./docs/architecture.md)
-  * Within docs, the versions of the api are stored in the appropriate folder under [versions](./docs/versions)
-  * The core API itself is detailed in the [spec file](./docs/vc-http-api.yml) and this spec should be used as the master reference for the API 
+  - Architecture for the issuer model is discussed in the [architecture document](./docs/architecture.md)
+  - Within docs, the versions of the api are stored in the appropriate folder under [versions](./docs/versions)
+  - The core API itself is detailed in the [spec file](./docs/vc-http-api.yml) and this spec should be used as the master reference for the API
 - [packages](./packages/) which contains test suites and related utilities for integration testing, test suite results, and related assets.
 
+### Data Visualization
+
+You may wish to programatically download the latest test results as JSON.
+
+#### Test Results for All Implementations
+
+- HTML Report: TODO
+
+```
+curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/testResults.json | jq
+```
```

Comment:
This nests the codeblock within the bullet

```suggestion
  ```
  curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/testResults.json | jq
  ```
```